wiki/People/Tournament_Committee/en.md

```diff
@@ -6,22 +11,33 @@ Their responsibilities include:
 
 - Managing reports from the [tournament reports form](https://pif.ephemeral.ink/tournament-reports)
 - Observing the general tournament scene and advising the [account support team](/wiki/People/Account_support_team) on a case-by-case basis
-- Regularly reviewing the [Official tournament support](/wiki/Tournaments/Official_support) page for revisions where necessary
+- Regularly reviewing the [Official tournament support](/wiki/Tournaments/Official_support) page for revisions when necessary
 
 The panel was first announced to the public on July 22, 2021 through the `#tournaments` channel in the [osu! Discord server](https://discord.com/invite/ppy) in response to the rapidly growing community tournament scene in osu!.
 
+## Contest Committee
+
+The **Contest Committee** is a subgroup directly associated with the **Tournament Committee** which is specifically dedicated to evaluating beatmapping contests.
+
+Their responsibilities include:
+
+- Observing the general contest scene and advising the [account support team](/wiki/People/Account_support_team) on a case-by-case basis
+- Regularly reviewing the [Official tournament support](/wiki/Contests/Official_support) page for revisions when necessary
+
+The panel was first announced to the public on January 27, 2024 through the `Mapping Discussion` forum.
```

Comment:
link

Reply:
fic

Reply:
the forum post

Reply:
what does this even mean?
(if anything I have tried searching the mapping discussion forum, no luck)

Reply:
there's supposed to be a forum post announcing this change but it's not up yet